### ISSUE TITLE
Found Python: error

### ISSUE THREAD

**Issue by soya-miruku** [losttech/BillionSongs#1]
Hey,
I really like this idea of having merge python and it's modules with .net
I have tried to start up this project but i am getting a weird error when i try to train the model
3.6-X64 at C:\Program Files\Python36\
   at Gradient.GradientSetup.GetEnvironment()
   at Gradient.GradientSetup.Initialize()
   at Gradient.GradientSetup.EnsureInitialized()
   at tensorflow.tf..cctor()
   --- End of inner exception stack trace ---
   at tensorflow.tf.no_op(String name)
   at Gradient.Samples.GPT2.Gpt2Program.Main(String[] args) in C:\Users\{USERNAME}\source\repos\BillionSongs\External\Gradient-Samples\GPT-2\GPT2Program.cs:line 12

any idea what could be causing this?

thanks in advance

**Comment by lostmsu** [comment]
The error message appears to be incomplete. Exception class and majority of the message are missing.

If I remember correctly, when you don't have TensorFlow for Python installed, you'll get this message.

If you have a NVidia GPU, you must install `tensorflow-gpu==1.10.*` with corresponding CUDA and CuDNN.
Otherwise (or if you don't want to deal with GPU), just install `tensorflow==1.10.*`

Other versions of TensorFlow might work too, but not yet supported.

For more details, see https://www.tensorflow.org/install/

**Comment by lostmsu** [comment]
Closing, as there was no response for issue details.
